mysql-test/suite/encryption/t/encrypt_and_grep.test

```diff
@@ -121,12 +122,12 @@ ALTER TABLE t6 ENCRYPTION='KEYRING', ALGORITHM=INPLACE;
 
 SET GLOBAL default_table_encryption = ONLINE_FROM_KEYRING_TO_UNENCRYPTED;
 
---echo # Only three tables should stayed encrypted - the ones with explicite encryption
+--echo #Three tables should stayed encrypted - the ones with explicit encryption
 --let $wait_timeout= 600
 --let $wait_condition=SELECT COUNT(*) = 3 FROM INFORMATION_SCHEMA.INNODB_TABLESPACES_ENCRYPTION WHERE MIN_KEY_VERSION = 1
 --source include/wait_condition.inc
 
---let $assert_text= Three tables should stayed encrypted - the ones with explicite encryption t1, t4 and t6
+--let $assert_text=Three tables should stayed encrypted and tablespace - the ones with explicit encryption t1, t4 and t6
```

Comment:
"Three tables should stay encrypted the tablespace should have encryption flag set?"

Reply:
done